test/sets.jl

```diff
@@ -1,5 +1,9 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+# OffsetVector is used to test `unique!` on non-standard indices (PR #20619).
+isdefined(Main, :TestHelpers) || (@eval Main include(joinpath(dirname(@__FILE__),
+   "TestHelpers.jl")))
```

Comment:
Woops, only just read this, if I try it inside `if isdefined...`, then the module doesn't get loaded outside the scope of the `if` block.